scripts/api-app-setup.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+cd $(dirname -- $0)
+
+echo Building api app...
+docker build -t api-app ../api/
+if (( $? )); then
+  echo Could not build api app, exiting...
+fi
+
+api_app_container="api_app"
+echo Removing running api app, if any...
+docker kill $api_app_container 1>/dev/null 2>/dev/null
+if (( $? )); then
+  echo Not running.
+fi
+docker container rm $api_app_container 1>/dev/null 2>/dev/null
+
+echo Checking envs...
+mongo_envs=(
+  "MONGO_URI_STR"
+  "MONGO_DB"
+  "MONGO_COLLECTION"
+)
+mongo_envs_arg=""
+for mongo_env in "${mongo_envs[@]}"
+do
+  if [ -v $mongo_env ]; then
+    mongo_envs_arg="$mongo_envs_arg -e $mongo_env"
+  else
+    echo "$mongo_env env var is not set! Using default"
+  fi
+done
+
+app_port=44440
+if [[ ! -v API_APP_PORT ]]; then
+  echo API_APP_PORT not set, defaulting to $app_port
+else
+  app_port=$API_APP_PORT
+fi
```

Comment:
You can do the same with a single line
```suggestion
app_port=${API_APP_PORT:-44440}
```

Reply:
Ok, using default arg for the echo as well.